src/components/list/_list.scss

```diff
@@ -103,15 +106,13 @@ $include-html: false !default;
     @include mint-list-basic-styles();
     display: inline-block;
     vertical-align: inherit;
-    font-size: $breadcrumbListFontSize;
-    line-height: $breadcrumbListFontSize * 1.5;
     color: $breadcrumbListSeparatorColor;
 
     &__element {
       display: inline-block;
 
       &:after {
-        content: '·';
+        content: '•';
```

Comment:
IMO it's wrong, but merge away, I'll fix that after I talk with Asia - https://github.com/brainly/style-guide/issues/282 .